docs/docs/modeling.md

```diff
@@ -106,6 +106,46 @@ classDiagram
 
 ## ShareHouse - シェアハウス
 
+The owner can register the House owned by the owner.
+
+### ShareHouse ID (Identifier)
+
+ShareHouse ID is the UUID.
+
+### ShareHouse Name
+
+The name of the share house can set and changed. The shareHouse has the default name when the shareHouse created without specifying the name.
+
+Constraints：
+
+- At least 1 character and no more than 15 characters.
+- New ShareHouses are listed in ascending order.
+
+### rotationCycle
+
+It can select Weekly or Fortnightly.
+
+Constraints：
+
+- Tenants cannot be assigned to a Category until the rotationCycle is set.
+
+### assignmentSheet
+
+Tenant's task assignment table.
+
+### List[Category]
+
+Category listings have multiple categories.
+
+Constraints：
+
+- All tenants must be assigned a category.
+- The same tenant cannot be selected.
```

Comment:
The constraints for `List[Category]` might not need to be written in the `ShareHouse`.